content/en/docs/concepts/containers/images.md

```diff
@@ -67,12 +67,8 @@ these values have:
 : the image is pulled only if it is not already present locally.
 
 `Always`
-: every time the kubelet launches a container, the kubelet queries the container
-  image registry to resolve the name to an image
-  [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). If the kubelet has a
-  container image with that exact digest cached locally, the kubelet uses its cached
-  image; otherwise, the kubelet pulls the image with the resolved digest,
-  and uses that image to launch the container.
+: every time the kubelet launches a container, the kubelet queries the container image registry to resolve the name to an image
+  [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). The kubelet then pulls the image with the resolved digest, and uses that image to launch the container.
```

Comment:
It looks to me as if the kubelet instructs the container runtime to pull that image. In that case, the cacheing behavior also depends on your chosen container runtime.